### DISCUSSION TITLE
Return current value of affection

### DISCUSSION THREAD

**Discussion by purplelunamine** [Monika-After-Story/MonikaModDev#9284] [QA - code and content creation (technical)] [answered]
If I'm writing a dialogue, how can I make Monika show her current affection value? Just the number, not the "name" of her current feeling.

**Comment by Iceninja64** [comment]
She doesn't say because she is not a robot and the mods didn't intend for that to be a question plus you are going to have to go to the affection logs if you want to know your actual affection values

**Comment by Booplicate** [comment, marked as answer]
There's no public accessor for affection.
